olmo/eval/downstream.py

```diff
@@ -9,11 +9,62 @@
 from sklearn.metrics import f1_score
 from torchmetrics import Metric
 
+from olmo.torch_util import barrier, get_fs_local_rank
+from olmo.util import load_dataset_from_disk, save_dataset_to_disk
+
 from ..tokenizer import Tokenizer
 
 log = logging.getLogger(__name__)
 
 
+def load_dataset(path, name, split, datasets_cache_dir: Optional[str] = None):
```

Comment:
nit: missing type hints
```suggestion
def load_dataset(path, name, split, datasets_cache_dir: Optional[str] = None):
```

Reply:
[c55974d](https://github.com/allenai/OLMo/pull/623/commits/c55974d5581d0cf44e86e996c7ec4a86b97d7966)